contessa/consistency_checker.py

```diff
@@ -39,15 +42,23 @@ def run(
         left_check_table: Dict,
         right_check_table: Dict,
         result_table: Dict,
+        columns: Optional[List[str]] = None,
+        time_filter: str = TIME_FILTER_DEFAULT,
+        left_custom_sql: str = None,
```

Comment:
I understand the current need of this. I think that this should not be needed in the end, or in extreme cases. Most of the dq checks should be buildable using the rules and e.g. passing it here. Just a note